### PR TITLE
Adjust fees slightly higher

### PR DESCRIPTION
**Pull request by TronBlack**

### CODE REVIEW

File: src/wallet/wallet.h [hunk 1]
```diff
@@ -46,13 +46,13 @@ static const unsigned int DEFAULT_KEYPOOL_SIZE = 1000;
 //! -paytxfee default
 static const CAmount DEFAULT_TRANSACTION_FEE = 0;
 //! -fallbackfee default
-static const CAmount DEFAULT_FALLBACK_FEE = 20000;
+static const CAmount DEFAULT_FALLBACK_FEE = 2000000;
```
Comment (lefty313) [inline src/wallet/wallet.h]: 100x increase does not seem to be `slightly higher` 💯

**Comment by bedri** [comment]
100x increase may cause a panic and increase in sell orders which will
cause a serious dump in the RVN price. Maybe it's better to increase it
step by step in a couple of minor upgrades.

**Comment by AydinAdn** [comment]
Whats the point of this? If I enjoyed high transaction fees, I'd use a bank

**Comment by russkidooski** [comment]
Current transactions fees per KB in USD as of july 2nd 2018: 0.00000015894$
Transaction fees after update: 0.000015894$
still dirt cheap

**Comment by russkidooski** [comment]
i wouldn't see why not.

**Comment by lefty313** [comment]
@TronBlack would be cool to get your opinion since you are author of this pull request

**Comment by TronBlack** [comment]
This has generated discussion.   
Just my opinion:  I think early on, fees are only needed for DDOS protection, and as a market-based solution for reducing excessive load on the chain.   For at least the first 12 to 20 years (3 to 5 block reward halvings), the block reward is enough to incent miners.   Bitcoin operated just fine with no minimum fee required.   But an easy way to attack a coin is to send junk transactions, and with no cost, that can be dangerous.

When adding assets into the mix, I think a fee prevents some potential abuses of the Ravencoin blockchain.

This is not setting the fee, but a default fallback fee, and for that reason, I'm open to lowering it.   Under load, the dynamic fee system inherited from bitcoin will kick in and require a higher fee.

The goal was not to have a much higher fee, but rather to bring it in line with the economics of having 21 Billion Ravencoin instead of 21 Million Bitcoin.

**Comment by TronBlack** [comment]
There was enough push-back on this particular change, that we are going to leave the existing min fee alone.   

The system will automatically suggest higher fees if the mempool gets full  (high-load conditions).   Bitcoin ran just fine for years with no fee required.   Some chance of a lower cost DDOS type attack.  It will cost something, but at a low cost.  We, as a team, decided this was not necessary, and we want to encourage the use of Ravencoin assets with very low fees.

For these reasons, I'm removing this PR.

**Pull request closed without merging**